modules/behavioural_qc/php/incomplete.class.inc

```diff
@@ -0,0 +1,151 @@
+<?php declare(strict_types=1);
+
+namespace LORIS\behavioural_qc;
+
+use LORIS\behavioural_qc\Provisioners\IncompleteProvisioner;
+use LORIS\Data\Filters\HasAnyPermissionOrUserSiteMatch;
+use \Psr\Http\Message\ServerRequestInterface;
+use \Psr\Http\Message\ResponseInterface;
+use \LORIS\Middleware\ETagCalculator;
+
+/**
+ * The Incomplete Class.
+ *
+ * GET request method for endpoint in behavioural_qc module.
+ * Used for populating the Incomplete Forms table.
+ *
+ * @category Behavioural
+ * @package  Loris
+ * @author   Alizée Wickenheiser <alizee.wickenheiser@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
```

Comment:
We only use the `@license` tag now.

Reply:
@driusan do you mean every other tag (ex. author) should be removed?